src/time_integration/methods_SSP.jl

```diff
@@ -218,6 +218,11 @@ function set_proposed_dt!(integrator::SimpleIntegratorSSP, dt)
     integrator.dt = dt
 end
 
+# used by adaptive timestepping algorithms in DiffEq
+function get_proposed_dt(integrator::SimpleIntegratorSSP)
+    return integrator.dt
+end
```

Comment:
```suggestion
```
Is that something new? In `main`, there is already a version of this routine with parameter `dt` (directly above):
```
# used by adaptive timestepping algorithms in DiffEq
function set_proposed_dt!(integrator::SimpleIntegratorSSP, dt)
    integrator.dt = dt
end
```
Do you really need this new one or did you just forget about it and had no problems because the other one already exists?

Reply:
This is a different routine. While `set_proposed_dt!` sets the time-step size to a value, `get_proposed_dt` just returns the value of `dt` that was computed previously. This is needed by the `GlmSpeedCallback`. 

Reason: The GLM-MHD equations have a parameter `c_h`, which is known as the hyperbolic divergence cleaning speed. The larger `c_h` is, the more effective the divergence cleaning is. However, since `c_h` is an advection speed, the larger it gets, the smaller the maximum time-step size gets for explicit time steppers. When using explicit time-integration schemes, `c_h` is usually adjusted at each time step (with a callback) such, that its value is as large as possible (to maximize the divergence cleaning effect) but not large enough to affect `dt`.

Reply:
Ah wow, sorry. Sometimes reading can help:sweat_smile: